core/modules/field/field.install

```diff
@@ -117,3 +117,25 @@ function field_update_1002() {
     $role_config->save();
   }
 }
+
+/**
+ * Check if the contrib module options_element is present, and uninstall it if
+ * so.
+ */
+function field_update_1003() {
+  // If the module is present on the site, it is already disabled (because it is
+  // included in backdrop_merged_modules()), so we cannot use module_exists().
```

Comment:
I don't think it is disabled per se, is it? It's just preventing the install file from loading. Regardless it might be safer to just do a direct db query to check if a row exists with name = options_element. Then you can output the message still.